tests/boulder-integration.sh

```diff
@@ -217,3 +220,5 @@ if ${CI:-false} || type nginx;
 then
     . ./certbot-nginx/tests/boulder-integration.sh
 fi
+
+coverage report --fail-under 52 -m
```

Comment:
We agreed that the better solution is probably to merge the coverage and py27 integration travis tests, and get a single threshold for unit + integration tests that way. But this is fine for now.